packages/entity-list/src/components/SearchFilterList/SearchFilterList.js

```diff
@@ -22,16 +32,32 @@ const SearchFilterButton = ({setActive, active, label}) =>
         e.stopPropagation()
       }}
       icon={active ? 'minus' : 'plus'}
-      dense />
+      dense/>
+    <StyledMenuWrapper active={active}>
+      <BallMenu buttonProps={{icon: 'ellipsis-h'}}>
+        {canEdit && <MenuItem onClick={() => window.open(`/e/Search_filter/${primaryKey}`, '_blank')}>
```

Comment:
use navigationStrategy here. We dont want to hardcode the admin routing inside entity-list